### ISSUE TITLE
`list/table` only shows one method

### ISSUE THREAD

**Issue by baggepinnen** [tkf/DocumentationOverview.jl#20]
I've tried the following
```julia
DocumentationOverview.table_md(
    DocumentationOverview.PublicAPI.of(RobustAndOptimalControl);
    signature = :strip_namespace,
    apicolumn = "Name",
)
```
while produces a nice looking table, but some functions in `RobustAndOptimalControl` contains several different methods that all have docstrings, and for those, only one of the docstrings is represented. Would it be possible to make all documented methods appear in the list/table?

Another question, I called `DocumentationOverview.PublicAPI.of(RobustAndOptimalControl)` in order not to included non-exported symbols in the list, which they otherwise were. Perhaps a keyword argument `nonexported = false` to `table/list` would make sense?

**Comment by tkf** [comment]
Yeah, it sounds like a good idea to show only exported names by default. We can use `hasdoc_and_exported(api) = api.hasdoc && api.exported` as the default value of the `include` option.

**Comment by tkf** [comment]
> Would it be possible to make all documented methods appear in the list/table?

It's possible that I screwed up implementing how docstrings are merged. I looked at how Base.Docs does it and guessed how it worked. It's done in

https://github.com/tkf/DocumentationOverview.jl/blob/4bf98c0fb3311f61a9b835577cd5193ff06c39c1/src/pkgapi.jl#L44-L52

The _intention_ was to accumulate all the docstrings defined in all submodules under `api.module`. (So, it doesn't show docstrings from, e.g., other packages.)
